bruno_first.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: bruno_first
+  type: http
+  seq: 1
+}
+
+get {
```

Comment:
It would be beneficial to add error handling or a timeout to the API call to ensure the test doesn't hang if the API is unresponsive. [important]